src/main/jbake/content/blog/2017/07/grafana-new-query-interface.adoc

```diff
@@ -0,0 +1,177 @@
+= Grafana: new query interface
+Joel Takvorian
+2017-07-20
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, metrics, grafana
+:figure-caption!:
+
+There has been several improvements lately in our Hawkular Grafana datasource, that is worth writing some words about.
+
+First, the way to query metrics by tags has changed since plugin link:https://github.com/hawkular/hawkular-grafana-datasource/releases/tag/v1.0.8[v1.0.8].
```

Comment:
This "First, second(ly)" looks odd (and Secondly even more :-)

I don't have a good idea on how to rephrase. you could make it a bullet-list with those 2 entries and drop 1st/2nd